hw2/03-count.js

```diff
@@ -1,3 +1,18 @@
 // Add your code here
+const input = document.getElementById('inText');
+const content = document.getElementById('text-content');
 
-input.addEventListener('keydown', handleKeyDown);
+function handleKeydown(event) {
+  // retrieve search term from input
+  const search = event.target.value.trim();
+  if (search === '') {
+    content.innerHTML = content.textContent;
+    return;
+  }
+  // retrieve the content to be searched
+  const { textContent } = content;
+  // find and highlight all occurrences of the search term in the text content
+  const highlightedTxt = textContent.replace(new RegExp(search, 'gi'), (match) => `<mark>${match}</mark>`);
```

Comment:
We don't want to highlight matches that are inside other words, just whole word matches. You can do this by adding word boundary anchors(`\b`), around the input, to your regex.